src/components/dev-hub/podcast-player/player-listen-on.js

```diff
@@ -0,0 +1,94 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import Button from '~components/dev-hub/button';
+import {
+    AppleLogo,
+    GoogleLogo,
+    SpotifyLogo,
+} from '~components/dev-hub/icons/audio-player';
+
+import { colorMap, screenSize, size } from '~components/dev-hub/theme';
```

Comment:
For the `colorMap` uses, could we use the emotion theme we have going and do something like this in the `styled`?

```
${({ theme }) => theme.colorMap.greyLightThree}
```

This lets us use the theme we have throughout the app and can keep these components a bit more general for if/when we change our color palette